atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/MonotonicChecker.java

```diff
@@ -47,7 +47,9 @@ public void visit(OkEvent event) {
 
             if (latestEventPerProcess.containsKey(process)) {
                 OkEvent previousEvent = latestEventPerProcess.get(process);
-                if (event.value() <= previousEvent.value()) {
+                Long previousTimestamp = Long.parseLong(previousEvent.value());
+                Long timestamp = Long.parseLong(event.value());
```

Comment:
`Long.parseLong()` returns a `long` which we can use for comparison, so there's no need to box it back into a `Long` - we'd save up to 2 object creations per event.

Also we'd be parsing each event's value twice (once when visiting it, and once when reading it as a previousEvent). I'd say readability wins here though, the alternatives I have in mind such as a parallel map of process ID to timestamp would be unnecessarily complex.